### DISCUSSION TITLE
Recognize target according to ID using Mysql + JavaScript

### DISCUSSION THREAD

**Discussion by kanzaky** [hiukim/mind-ar-js#305] [Q&A] [answered]
I've just implemented a Dashboard using javascript and now i would like to add an option to create a new target (in my case to create a new event card information) and that target will be link to the ID from that target on the database, but how can my WebAR use all those targets on the database as imageTargetSrc - it's a different form of multi-target im trying to implement - i will be more that open to share my final product in the end, its a school task i was told i wouldn't be able to do xD and i need to prove them wrong

![image](https://user-images.githubusercontent.com/35463515/204879223-bf1a9aa8-eadd-43a7-b866-f8838b17b006.png)

**Comment by LeoVC1** [comment]
Did you manage to get this working? I'm trying to create the same

**Comment by BostonLeeK** [comment, marked as answer]
I try understand. You want to change tracking from index of files in bundle to check of id of marker? If yes, you need to make changes to converter (add field of marker id to object) and then edit mindar main lib file change index check to id check.
